css/index.css

```diff
@@ -1,3 +1,395 @@
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+/* * {
+    box-sizing: border-box;
+} */
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+body {
+	width: 877px;
+	margin: 0 auto;
+}
+
+p {
+	line-height: 1.25em;
+}
+
+header {
+	display: flex;
+	justify-content: center;
+	height: 80px;
+	width: 100%;
+	background: white;
+	position: fixed;
+	top: 0;
+	left: 0;
+	
+}
+
+.wrapper {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	/* width: 877px; */
+}
+
+nav {
+    display: flex;
+    justify-content: space-between;
+    width: 600px;
+
+}
+nav a {
+	text-decoration: none;
+	color: lightgrey;
+	font-size: 16px;
+	width: 16%;
+
+}
+
+.logo{
+	display: flex;
+	justify-content: flex-end;
+	width: 277px;
+}
+.hero {
+	margin: 125px 0 75px;
+}
+
+.title {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	text-align: center;
+	margin-top: 65px;
+	
+}
+
+.title h1 {
+	font-size: 80px;
+}
+
+.title a {
+	border: 1px solid black;
+	padding: 0px 30px;
+	text-decoration: none;
+	color: black;
+	font-size: 20px;
+
+}
+
+/* main section /////////////////*/
+main {
+	border-top: 1px solid black;
+	border-bottom: 1px solid black;
+}
+
+main, .middle-img {
+	padding: 40px 0;
+
+}
+
+.half div, .thirds div {
+	white-space: normal;
+	display: inline-flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	/* height: 100%; */
+	vertical-align: top;
+}
+
+.half {
+	white-space: nowrap;
+	
+}
+
+.half div   {
+	width: 50%;
+}
+
+.thirds div {
+	width: 33%;
+}
+
+
+.row {
+	display: flex;
+	justify-content: space-between;
+	margin-bottom: 40px;
+}
+
+/* footer //////////////////////// */
+footer {
+	margin-top: 40px;
+}
+
+footer a {
+	display: flex;
+	text-decoration: none;
+	color: black;
+}
+
+.phone {
+	margin: 24px 0;
+	
+}
+
+#copyright {
+	text-align: center;
+	margin: 48px 0 108px;
+	font-size: 14px;
+}
+/* //////////////////////////////// */
+@media all and (max-width: 800px) {
+    header nav {
+     order: 2;
+     width: 800px;
+     justify-content: space-between;
+     margin-left: 45px;
+ }
+
+ .wrapper {
+     flex-direction: column;
+ }
+
+.logo{
+    display: flex;
+    justify-content: center;
+    order: 1;
+    width: 277px; 
+}
+
+#code-snip {
+    display: none;
+    }
+    
+ .half div.description {
+    margin: 40px auto;
+    flex-wrap: wrap;
+    width: 350px;
+}
+
+.half div.title  {
+    display: block;
+    width: 800px;
+}
+.title {
+    margin-top: 150px;
+    margin-left: -10px;
+}
+
+.hero {
+}
+
+main {
+    margin: 0 40px;
+    width: 700px;
+}
+
+main p {
+    font-size: 0.85em;
+}
+
+.thirds div.description {
+    width: 200px;
+    margin-right: 30px;
+}
+
+main .middle-img {
+    width: 700px;
+    margin-top: -60px;
+}
```

Comment:
Avoid negative margins. White space does not appear from nowhere. Use the inspector tools to track down the source of the space and address it directly. I know there was some confusing discussion on slack regarding negative margins, but the general consensus is negative margins are bad practice.

Reply:
Yeah, I kind of figured it wasn't best practice and honestly couldn't figure out why my margins were so wonky, it was something I did to make it work for submission. I will not be doing this going forward!